test/e2e/config/azwi.yaml

```diff
@@ -0,0 +1,272 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/audit-version: latest
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/warn: restricted
+    pod-security.kubernetes.io/warn-version: latest
+  name: azure-workload-identity-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-admin
+  namespace: azure-workload-identity-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-role
+  namespace: azure-workload-identity-system
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-role
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - admissionregistration.k8s.io
+    resources:
+      - mutatingwebhookconfigurations
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-rolebinding
+  namespace: azure-workload-identity-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: azure-wi-webhook-manager-role
+subjects:
+  - kind: ServiceAccount
+    name: azure-wi-webhook-admin
+    namespace: azure-workload-identity-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: azure-wi-webhook-manager-role
+subjects:
+  - kind: ServiceAccount
+    name: azure-wi-webhook-admin
+    namespace: azure-workload-identity-system
+---
+apiVersion: v1
+data:
+  AZURE_ENVIRONMENT: AzurePublicCloud
+  AZURE_TENANT_ID: b39138ca-3cee-4b4a-a4d6-cd83d9dd62f0
```

Comment:
why is there data for TENANT_ID checked in here?

Reply:
That is a miss. Will remove that.